### ISSUE TITLE
open source

### ISSUE THREAD

**Issue by luluhoc** [cryptoton/coin-listing#1]
Where are all the files for the bot?
You've stated on your website that it is open source can you share the code?
The bot is not working

**Comment by sirkbitt** [comment]
Yes, girhub project is equal to open source. Maybe the dev anstak could answer
